src/language/branches.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Branches 
+description: Learn how to use branches to control the flow of your Dart code.
+---
+
+You can control the flow of your Dart code using branches:
+
+-   `if`, `case`, and `else`
+-   `switch` and `case` statements
+-   `switch` and `case` expressions
```

Comment:
`else` is always contained and subordinate to an `if` statement/element. Likewise, `case` is always part of some surrounding `switch` statement. I wouldn't refer to them as statements or expressions on their own since they really aren't. Maybe:

```
You can control the flow of your Dart code using branching:

-  `if` statements and elements
-  `if-case` statements and elements
-   `switch` statements and expressions
```

And then just describe `else` and `case` as part of their respective contained things.